src/dft/backends/mklcpu/backward.cpp

```diff
@@ -39,66 +39,70 @@ namespace mklcpu {
 
 //In-place transform
 template <typename descriptor_type, typename data_type>
-ONEMKL_EXPORT void compute_backward(descriptor_type &desc, sycl::buffer<data_type, 1> &inout) {
+ONEMKL_EXPORT void compute_backward(descriptor_type& /*desc*/,
+                                    sycl::buffer<data_type, 1>& /*inout*/) {
     throw mkl::unimplemented("DFT", "compute_backward", "Not implemented for MKLCPU");
 }
 
 //In-place transform, using config_param::COMPLEX_STORAGE=config_value::REAL_REAL data format
 template <typename descriptor_type, typename data_type>
-ONEMKL_EXPORT void compute_backward(descriptor_type &desc, sycl::buffer<data_type, 1> &inout_re,
-                                    sycl::buffer<data_type, 1> &inout_im) {
+ONEMKL_EXPORT void compute_backward(descriptor_type& /*desc*/,
```

Comment:
whats the need for `/*desc*/` ?

Reply:
This is one way to fix unused variable warnings. The advantage is that it still gives the developer the original name of the variable if we decide to use it later.